### PR TITLE
Update `kratos_endpoints` interface

### PR DESCRIPTION
**Pull request by natalian98**
This PR updates the project-internal `kratos_endpoints` interface as it will provide more endpoints due to integration with Oathkeeper.

### CODE REVIEW

**Comment by natalian98** [comment]
@PietroPasotti could you check this PR and assign more reviewers if needed? Thanks a lot

File: interfaces/kratos_endpoints/v0/README.md [hunk 1]
```diff
@@ -25,8 +25,8 @@ Both the requirer and the provider need to adhere to a certain set of criteria t
 
 ### Provider
 
-- Is expected to serve admin and public API endpoints 
-- Is expected to write the public and admin URLs to the application databag.
+- Is expected to serve admin and public API endpoints
+- Is expected to write the public, admin, login and sessions URLs to the application databag.
```
Comment (PietroPasotti) [inline interfaces/kratos_endpoints/v0/README.md:29]:
```suggestion
- Is expected to write the public, admin, login and sessions endpoint URLs to the application databag.
```

**Review by PietroPasotti — APPROVED** [review]
LGTM!

**Comment by natalian98** [comment]
@simskij mind having a look? It needs 2 approvals.

**Review by simskij — APPROVED** [review]

**Pull request merged**